lib/observe.js

```diff
@@ -0,0 +1,74 @@
+var _ = require('lodash');
+var merge = require('viewdb').merge;
+var MongoOplog = require('mongo-oplog');
+
+var logger = {
+  added: function() { console.log("added: "); console.log(arguments); },
+  removed: function() { console.log("removed: "); console.log(arguments); },
+  changed: function() { console.log("changed: "); console.log(arguments); },
+  moved: function() { console.log("moved: "); console.log(arguments); },
+};
+
+var oplogUri = 'mongodb://127.0.0.1:27017/local';
```

Comment:
kan man inte skica in en mongodb instans?
if (typeof uri !== 'string') {
    if (uri && uri.collection) {
      db = uri
      connected = true
    } else {
      throw new Error('Invalid mongo db.')
    }
  }